### DISCUSSION TITLE
Double Commander sees many mount points when firejailed

### DISCUSSION THREAD

**Discussion by tredondo** [netblue30/firejail#4723] [Q&A]
I'm trying to run [Double Commander](https://github.com/doublecmd/doublecmd). But even with `--noprofile`, Double Commander looks like this:

![image](https://user-images.githubusercontent.com/10043369/143889041-b89d5e3e-1b79-4b22-977a-68db2be9f2b5.png)

Normally, it should look like this:

![image](https://user-images.githubusercontent.com/10043369/143889153-f72ddc49-f255-4673-9422-ab1f89f4c5f3.png)

**Comment by rusty-snake** [comment]
For me:
```
firejail --quiet --noprofile wc -l /proc/mounts
677 /proc/mounts
```
